src/component/TopBar.jsx

```diff
@@ -4,32 +4,36 @@ import PropTypes from 'prop-types';
 import {
   AppBar, Typography, Toolbar, Switch, Icon, Hidden, Box, Button,
 } from '@material-ui/core';
+import { useHistory } from 'react-router-dom';
 import SpacingDesign from './context/design/SpacingDesign';
 
-const TopBar = ({ darkMode, setDarkMode }) => (
-  <AppBar position="sticky">
-    <Toolbar>
-      <Hidden mdUp>
-        <Box style={{ flexGrow: 0.25 }} />
-      </Hidden>
-      <Box style={{ flexGrow: 1 }} align="center">
-        <Button style={SpacingDesign.marginLeft(5)} aria-label="Go to home screen">
-          <Icon className="fa fa-cloud-meatball" color="action" />
-          <Hidden mdDown>
-            <Typography
-              style={{ ...SpacingDesign.marginx(3) }}
-              color="textSecondary"
-            >
-              Whats for dinner
-            </Typography>
-          </Hidden>
-        </Button>
-      </Box>
-      <Switch check={darkMode.toString()} onChange={() => setDarkMode(!darkMode)} color="secondary" aria-label="dark mode toggling" />
-    </Toolbar>
-  </AppBar>
-);
+const TopBar = ({ darkMode, setDarkMode }) => {
+  const history = useHistory();
```

Comment:
Have we been consistent throughout with useHistory? I know that I've seen Link components elsewhere. I think the Link automatically pushes an entry to the history stack, so would it be better to stay consistent and just keep using Links?

Reply:
I've been using useHistory on things when it's a button with typography inside once in a while, because it changes the font to an "a" tag styling. I could possibly use the link component and change the styling. I'll check it out if doing Link component will change the styling